src/neofs_testlib/cli/cli_command.py

```diff
@@ -65,6 +65,11 @@ def _format_command(self, command: str, **params) -> str:
     def _execute(self, command: Optional[str], **params) -> CommandResult:
         return self.shell.exec(self._format_command(command, **params))
 
+    def _execute_interactive(self, patterns: list, inputs:list, command: Optional[str], **params) -> CommandResult:
```

Comment:
why we need separate patterns and inputs if we still need to verify count of them?
it's better to pass `inputs: list[InteractiveInputs]` directly here and in bellow method.